packages/sdk-react/src/core/theme/types.ts

```diff
@@ -0,0 +1,219 @@
+import { components, Services } from '@/api';
+import { ChipProps, SelectProps } from '@mui/material';
+
+export type ThemeConfig = {
+  borderRadius?: number;
+
+  colors?: {
+    primary?: string;
+    secondary?: string;
+    neutral?: string;
+
+    background?: string;
+
+    text?: string;
+  };
+
+  typography?: {
+    fontFamily?: string;
+    fontSize?: number;
+
+    h1?: {
+      fontSize?: string | number;
+      fontWeight?: string | number;
+      lineHeight?: string | number;
+    };
+    h2?: {
+      fontSize?: string | number;
+      fontWeight?: string | number;
+      lineHeight?: string | number;
+    };
+    h3?: {
+      fontSize?: string | number;
+      fontWeight?: string | number;
+      lineHeight?: string | number;
+    };
+    subtitle1?: {
+      fontSize?: string | number;
+      fontWeight?: string | number;
+      lineHeight?: string | number;
+    };
+    subtitle2?: {
+      fontSize?: string | number;
+      fontWeight?: string | number;
+      lineHeight?: string | number;
+    };
+    body1?: {
+      fontSize?: string | number;
+      fontWeight?: string | number;
+      lineHeight?: string | number;
+    };
+    body2?: {
+      fontSize?: string | number;
+      fontWeight?: string | number;
+      lineHeight?: string | number;
```

Comment:
There's a lot of repetition here, can we explore creating a type for our typography and reusing it?

Reply:
agree. fixed